src/Core/Content/ImportExport/DataAbstractionLayer/Serializer/Entity/MediaSerializer.php

```diff
@@ -63,6 +64,10 @@ public function deserialize(Config $config, EntityDefinition $definition, $entit
             return $deserialized;
         }
 
+        if (!Feature::isActive('v6.8.0.0')) {
+            $url = str_replace(' ', '%20', $url);
```

Comment:
Is that really everything? Don't we have sanitizers or encoders who do this without hardcoded replacements?

Reply:
Actually no, that just fixes the space issue. But we should do proper encoding of the path in the URL here, like we do in other places.